### PR TITLE
Feat: component faq v2

### LINKED ISSUE #81: Component: FAQ
tailwind ui or shadcn accordion

### PR DESCRIPTION
**Pull request by IounOm**
## What does it do?
- edit FAQ component from comment in PR #114

Additional code from PR [#114](https://github.com/Fork-It-Community/forkit.community/pull/114) in branch  [feat/component-faq](https://github.com/Fork-It-Community/forkit.community/tree/feat/component-faq)
- create FAQ component

## Screenshot
Screenshot 2567-10-08 at 18 09 59
Screenshot 2567-10-08 at 17 57 15

## Related issue
- close #81

### CODE REVIEW

File: src/components/faq.tsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    answer: string;
+  }[];
+};
+
+export const FAQ = (props: Readonly<FaqProps>) => {
+  return (
+    <div className="py-16" id="faq">
+      <article className="mx-auto max-w-3xl px-6 lg:px-8">
+        <h2 className="text-center font-heading text-3xl font-bold sm:text-4xl">
+          {props.title}
```
Comment (houssembaltii) [inline src/components/faq.tsx:21]: it's called FAQ (Frequently Asked Questions)  component, i don't think we will use another title except Frequently Asked Questions .

File: src/components/faq.tsx [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  AccordionTrigger,
+} from "@/components/ui/accordion";
+
+type FaqProps = {
+  title: string;
+  list: {
+    question: string;
+    answer: string;
+  }[];
+};
```
Comment (houssembaltii) [inline src/components/faq.tsx:14]:
```suggestion
  faq: CollectionEntry<"events">["data"]["faq"];
};
```

**Review by houssembaltii — CHANGES_REQUESTED** [review]
besides those two small comments , everything is ok to me !

File: src/components/faq.tsx (hunk 1 above)
Comment (IounOm) [reply src/components/faq.tsx]: fixed!

File: src/components/faq.tsx (hunk 2 above)
Comment (IounOm) [reply src/components/faq.tsx]: Fixed!

**Review by JeanneGrenet — APPROVED** [review]

**Review by houssembaltii — APPROVED** [review]

**Pull request merged**